qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/SnapshotWithExportersTest.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.clustering;
+
+import static io.camunda.zeebe.test.StableValuePredicate.hasStableValue;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.qa.util.actuator.PartitionsActuator;
+import io.camunda.zeebe.qa.util.testcontainers.ZeebeTestContainerDefaults;
+import io.camunda.zeebe.snapshots.impl.FileBasedSnapshotId;
+import io.camunda.zeebe.test.util.socket.SocketUtil;
+import io.zeebe.containers.ZeebeContainer;
+import io.zeebe.containers.ZeebeVolume;
+import io.zeebe.containers.exporter.DebugReceiver;
+import java.time.Duration;
+import java.util.stream.IntStream;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+
+final class SnapshotWithExportersTest {
+
+  @Test
+  void shouldIncludeExportedPositionInSnapshot() {
+    // given
+    try (final var debugReceiver =
+        new DebugReceiver((record) -> {}, SocketUtil.getNextAddress(), true).start()) {
+      try (final var zeebe =
+          new ZeebeContainer(ZeebeTestContainerDefaults.defaultTestImage())
+              .withDebugExporter(debugReceiver.serverAddress().getPort())) {
+        zeebe.start();
+
+        final var partitions = PartitionsActuator.of(zeebe);
+
+        try (final var client =
+            ZeebeClient.newClientBuilder()
+                .gatewayAddress(zeebe.getExternalGatewayAddress())
+                .usePlaintext()
+                .build()) {
+
+          publishMessages(client);
+          final var exporterPosition =
+              Awaitility.await("Exported position is stable")
```

Comment:
:thought_balloon: This isn't the first place we want to wait until the exporter has exported everything. I wonder if it might not be useful to add this to the partitions actuator, or some other actuator letting us introspect the current last exported positions :thinking: